aries_cloudagent/ledger/indy.py

```diff
@@ -1048,25 +1048,12 @@ async def accept_txn_author_agreement(
         storage = self.get_indy_storage()
         await storage.add_record(record)
         if self.pool.cache:
-            # TAA acceptance must be set for each wallet and pool combination
-            cache_key = (
-                TAA_ACCEPTED_RECORD_TYPE
-                + "::"
-                + self.wallet.opened.name
-                + "::"
-                + self.pool.name
-            )
+            cache_key = TAA_ACCEPTED_RECORD_TYPE + "::" + self.pool.name
```

Comment:
I think this is reverting a recent fix for the TAA caching

Reply:
Do need me to do anything here?